coordinator/tests/services/translation/incoming/release/release_success.xml

```diff
@@ -0,0 +1,459 @@
+<?xml version="1.0" encoding="UTF-8"?><hl7:PORX_IN070101UK31 xmlns:hl7="urn:hl7-org:v3">
```

Comment:
Do we have an example for this under models/examples?

Reply:
Not yet. This example is lifted from Spine's integration tests. Once we can get a real response back from Spine, I'll add the request / response pair in the usual place.

Reply:
Although I could just put this one in there for now?